crowdin.py

```diff
@@ -36,6 +36,11 @@ class bcolors:
     "time_short_24_hour",
     "time_short_12_hour"
 ]
+
+# List of languages to skip date format check for - e.g. in Japanese, it's legal to have Japanese letters as part of the date format string:
+# https://en.wikipedia.org/wiki/Date_and_time_notation_in_Japan#:~:text=The%20most%20commonly%20used%20date,%22Wednesday%2031%20December%202008%22.
+SKIP_DATE_FORMAT_CHECK = ['ja']
```

Comment:
Do we still need to do this if date format problems register as warnings and not errors? Skipping `ja` means a crash-inducing bug could sneak in via `ja`-locale date formats, and not failing on warnings means a crash-inducing bug could sneak in via any locale. IMO if we're not going to treat this as an error, we should just include `ja` in this check.

Reply:
This workaround specifically is just for date-format keys, not for the entire JA language code - so other keys in that language are checked (but not the date ones)